ci-operator/step-registry/upi/deprovision/heterogeneous/vpc/upi-deprovision-heterogeneous-vpc-commands.sh

```diff
@@ -86,16 +85,6 @@ function cleanup_ibmcloud_vpc() {
     fi
   done
 
-  echo "Cleaning up COS Instances"
-  VALID_COS=$(ic resource service-instances 2> /dev/null | grep "${cos_name}" || true)
-  if [ -n "${VALID_COS}" ]
-  then
-    for COS in $(ic resource service-instance "${cos_name}" --output json -q | jq -r '.[].guid')
-    do
-      retry "ic resource service-instance-delete ${COS} --force --recursive"
-    done
-  fi
-
```

Comment:
This might be useful since it cleans up the COS instances, which are used to load the coreos image into VPC.